package.json

```diff
@@ -20,9 +20,7 @@
     "babel-core": "^5.8.21",
     "bootstrap": "^3.3.4",
     "clear-cut": "^2.0.1",
-    "coffee-cash": "0.8.0",
```

Comment:
This is going to require a change in https://github.com/atom/timecop/blob/master/lib/cache-panel-view.coffee to get timecop working again.